src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-form/rbd-form.component.ts

```diff
@@ -543,6 +588,15 @@ export class RbdFormComponent extends CdForm implements OnInit {
     }
     if (this.mode === this.rbdFormMode.editing) {
       this.rbdForm.get('name').setValue(response.name);
+      if (response?.mirror_mode === 'snapshot' || response.features_name.includes('journaling')) {
```

Comment:
because mirror_mode says journal even when the journaling is disabled. I mean even if there is no mirroring enabled, the mode is always journal. so I couldn't rely on the mirror_mode for journal. I think I discussed this with you and you said it might be an issue from rbd side? @ideepika

Reply:
I see, yeah I remember now, since we use journaling feature just for mirroring, it should be fine, I will take a look into the discrepancy, please open a tracker if you happen to reproduce it. @nizamial09

Reply:
https://tracker.ceph.com/issues/55939, here's the tracker. Thanks to @Sarthak0702